savaslabs/savaslabs.github.io
# Issues #3183 and #3188: values pages

@oksana-c This PR does the following:

- Adds the key-value list item component and the key-value list region. This is used for the core values names + descriptions, but I wanted to make it reusable. 
- Removes the old mission and values page and adds a new one that loops through the values and prints them as key-value list item components
- Adds the "Move work forward" CTA to the mission and values page
- Adjusts some of the layout work I'd done previously. I want to be able to use the default layout for all pages and remove the page and page-narrow layouts. For this reason, instead of wrapping all of the page content in the `wrapper` class that adds some padding, I plan on wrapping each region in that class. This might be more info than you need but perhaps it will be useful! Basically I'm trying to make it easy to space things consistently.

## To test

- Check out the key-value list item in the Components section of the style guide and the key-value list region in the Regions section
- Visit localhost:3000/company/mission-and-values to see the WIP page
- You'll also notice that some of the home page/other page content is now too wide due to the `wrapper` class I refactored. This will all be fixed!
